app/src/main/java/at/roteskreuz/stopcorona/model/repositories/ExposureNotificationRepository.kt

```diff
@@ -148,7 +147,7 @@ class ExposureNotificationRepositoryImpl(
             return
         }
         registeringWithFrameworkState.loading()
-        exposureNotificationClient.start()
+        exposureNotificationClient.stop()
```

Comment:
This should have been a fix to the develop branch and not mixed in in here

Reply:
Moved to https://github.com/austrianredcross/stopp-corona-android/pull/85

Reply:
Updated PR description